test/functional/api/cas/ioctl/cas_structs.py

```diff
@@ -0,0 +1,108 @@
+#
+# Copyright(c) 2021 Intel Corporation
+# SPDX-License-Identifier: BSD-3-Clause-Clear
+#
+
+import ctypes
+from enum import Enum
+
+
+class RequestCode(Enum):
+    START_CACHE_CODE = ctypes.c_uint(21).value
```

Comment:
What does this construct do? I just checked that `type(c_uint(21).value) == int` so its equivalent with `START_CACHE_CODE = 21` (maybe aside from integer wraparound effects, but that doesn't seem to be needed here)